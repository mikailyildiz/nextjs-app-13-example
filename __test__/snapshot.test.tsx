import { render, screen } from '@testing-library/react'
import AddBookmark from '../app/[lang]/components/addBookmark'

it('renders addbookmark unchanged', () => {
  const {container} = render(
    <AddBookmark selected={false} onBookmark={() => {}} />
  )
  expect(container).toMatchSnapshot()
})