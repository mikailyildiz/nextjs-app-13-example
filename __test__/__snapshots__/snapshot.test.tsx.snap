// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders addbookmark unchanged 1`] = `
<div>
  <span
    class="bookmark "
    data-test="add-bookmark"
    title="Add Bookmark"
  >
    <svg
      aria-hidden="true"
      fill="none"
      stroke="currentColor"
      stroke-width="1.5"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M17.593 3.322c1.1.128 1.907 1.077 1.907 2.185V21L12 17.25 4.5 21V5.507c0-1.108.806-2.057 1.907-2.185a48.507 48.507 0 0111.186 0z"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
  </span>
</div>
`;
